engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schema/StringForcingDatumReader.scala

```diff
@@ -0,0 +1,47 @@
+package pl.touk.nussknacker.engine.avro.schema
+
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.avro.Schema
+import org.apache.avro.generic.{GenericData, GenericDatumReader}
+import org.apache.avro.io.Decoder
+import org.apache.avro.reflect.{ReflectData, ReflectDatumReader}
+import org.apache.avro.specific.{SpecificData, SpecificDatumReader}
+import org.apache.avro.util.Utf8
+
+import scala.util.{Properties, Try}
+
+trait StringForcingDatumReader[T] extends GenericDatumReader[T]  {
+  self: GenericDatumReader[T] =>
+
+  override def readString(old: Any, expected: Schema, in: Decoder): AnyRef = super.readString(old, expected, in) match {
+    case data: Utf8 if AvroStringSettings.forceUsingStringForStringSchema => data.toString
```

Comment:
why it is still needed if original implementation already check result of `findStringClass`?

Reply:
i'ts not needed anymore, I forgot to clean it up :)